salt/auth/.loader.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8; mode: python -*-
+
+'''Pre-loader for "auth" sub-system'''
+
+
+from __future__ import absolute_import
+
+from salt import loader_core
+from salt import loader_pre
+
+
+@loader_pre.LoaderFunc
```

Comment:
All of the old ``salt.loader.SUBSYSTEM()`` functions that were in ``salt/loader.py`` were moved into their respective sub-directories (viz the functions that instantiate a ``LazyLoader`` for each sub-directory).